FWCore/PythonFramework/test/runPythonTests.sh

```diff
@@ -12,4 +12,10 @@ do
 done
 
 echo "running test_producer.py"
-python ${LOCAL_TEST_DIR}/test_producer.py || die "test_producer.py failed"
+{ 
+python3 -c "from FWCore.PythonFramework.CmsRun import CmsRun" 2>/dev/null && python3 ${LOCAL_TEST_DIR}/test_producer.py || die "test_producer.py failed"
```

Comment:
what will happen is `python3 ${LOCAL_TEST_DIR}/test_producer.py` failed for some reason? I think in this case it is going to run the next part of the code and will try to run it using python2. We do not want this. I think a simple logic something like the following is enough in this case
```
if python3 -c "from FWCore.PythonFramework.CmsRun import CmsRun" 2>/dev/null
 ; then
  python3 ${LOCAL_TEST_DIR}/test_producer.py || die "test_producer.py failed"
else
  python2 ${LOCAL_TEST_DIR}/test_producer.py || die "test_producer.py failed"
fi
```

Reply:
Thanks @smuzaffar, I removed my `+1`. Probably better that you check this.